### PR TITLE
Maia Learn Documentation

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/refguide/modeling/mendix-ai-assistance/maia-learn.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: "Maia Learn"
+linktitle: "Learn"
+url: /refguide/maia-learn/
+weight: 5
+description: "Describes the features in Maia Learn."
+---
+
+## Introduction 
+
+Maia Learn helps you to quickly learn Mendix core concepts and get started with Studio Pro. It is a platform where you can navigate to different learning materials while providing you a holistic learning experience.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/maia-learn.md]:
```suggestion
Maia Learn helps you to quickly learn Mendix core concepts and get started with Studio Pro. It is a tab within the **Maia** pane where you can navigate to different learning materials while providing you a holistic learning experience.
```

File: content/en/docs/refguide/modeling/mendix-ai-assistance/maia-learn.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* On the **Welcome** page of Studio Pro, click **Open Maia Learn**
+* Or click the **Maia** ({{% icon name="sparkles" %}}) icon at the top right bar of Studio Pro and click **Learn**
+
+The learning interface appears at the right side of Studio Pro under the **Maia** tab:
+
+{{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/maia-learn/maia-learn-interface.png" width="300px">}}
+
+Click **Mendix Core Concepts**, you can start your learning journey with Maia Learn and learn all the basics about domain models, pages, logic, security, and integrations, including how you can apply these core concepts in Studio Pro.
+
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/maia-learn.md:25]: You can also access additional materials on the topic, such as related documentation or a learning path.

Comment (Yiyun333) [reply content/en/docs/refguide/modeling/mendix-ai-assistance/maia-learn.md:25]: Thanks! Added this information.

**Comment by Yiyun333** [comment]
Reviewed by PM as well. Can be published for 10.18.

**Pull request merged**